bcipy/orchestrator/parameters/actions.py

```diff
@@ -0,0 +1,85 @@
+from typing import List
```

Comment:
You will probably want to use the module `actions` for defining the `Action` classes. The functionality in here is really about configuration, so I would recommend renaming this as `config.py` or `configuration.py`. I also don't think this needs to be nested in a parameters directory. It can be a top-level module within orchestrator.